release/README.md

```diff
@@ -17,30 +17,28 @@ All of the above are generally freely available, e.g. installable via apt in Ubu
 
 ## Release Procedure
 
-First, check that the [latest commit](https://github.com/RaRe-Technologies/smart_open/commits/master) passed all CI.
-
-For the subsequent steps to work, you will need to be in the top-level subdirectory for the repo (e.g. /home/misha/git/smart_open).
-
-Prepare the release, replacing 2.3.4 with the actual version of the new release:
-
-    bash release/prepare.sh 2.3.4
-
-This will create a local release branch.
-Look around the branch and make sure everything is in order.
-Checklist:
-
-- [ ] Does smart_open/version.py contain the correct version number for the release?
-- [ ] Does the CHANGELOG.md contain a section detailing the new release?
-- [ ] Are there any PRs that should be in CHANGELOG.md, but currently aren't?
-
-If anything is out of order, make the appropriate changes and commit them to the release branch before proceeding.
-
-**This is the point of no return**.
-**Once you're happy with the release branch**, run:
-
-    bash release/merge.sh
-
-Congratulations, at this stage, you are done!
+> ![New GitHub Release](https://github.com/user-attachments/assets/cf8f2fa4-37c1-4e50-9fd8-ab6e3fd705b5)
+> *New GitHub Release dialog*
+
+- Check that the [latest commit](https://github.com/piskvorky/smart_open/commits/develop) on `develop` passed all CI.
+- Make sure you're on `develop` and you're up to date locally:
+    - `git checkout develop && git pull`
```

Comment:
Can we reduce the number of manual steps and copy/pasting here? Previously, releasing was a simple matter of calling prepare.sh and merge.sh and following the on-screen instructions, with occasional reference to this README.md.

Reply:
fdd7e2e 👍 

by leveraging GitHub's 'generate release notes' button, this flow eliminates the need for any clipboard action: the format in the github release will be slightly different than in CHANGELOG.md, but contains the exact same info and makes this flow as lean as possible!